x-pack/solutions/security/plugins/security_solution/public/management/components/endpoint_responder/lib/console_commands_definition.ts

```diff
@@ -530,7 +530,7 @@ export const getEndpointConsoleCommands = ({
       name: 'runscript',
       about: getCommandAboutInfo({
         aboutInfo: CROWDSTRIKE_CONSOLE_COMMANDS.runscript.about,
-        isSupported: doesEndpointSupportCommand('runscript'),
+        isSupported: agentType === 'crowdstrike' && doesEndpointSupportCommand('runscript'),
```

Comment:
How about performing this check inside `doesEndpointSupportCommand` since it already has access to `agentType`?

Reply:
Thanks, I've thought about it but then `doesEndpointSupportCommand` would have to check for agentType and command === 'runscript' . 
I think we will have to look closer to it and create a more generic check one day. 
Is that ok with you @szwarckonrad  ?